java/system-test/system-test-suite/src/test/java/sleeper/systemtest/suite/EmrPersistentBulkImportIT.java

```diff
@@ -41,7 +41,7 @@
 import static sleeper.systemtest.suite.testutil.PartitionsTestHelper.partitionsBuilder;
 
 @Tag("SystemTest")
-@Tag("slow")
+@Tag("expensive")
 public class EmrPersistentBulkImportIT {
```

Comment:
I don't think this is too expensive for the nightly functional test suite, since it only imports 100 records and the cluster shouldn't exist for very long.